Xamarin.AndroidBinderator/Engine.cs

```diff
@@ -198,6 +215,8 @@ static List<BindingProjectModel> BuildProjectModels(BindingConfig config, Dictio
 
 				var artifactDir = Path.Combine(config.BasePath, config.ExternalsDir, mavenArtifact.GroupId);
 				var artifactFile = Path.Combine(artifactDir, $"{mavenArtifact.ArtifactId}.{mavenProject.Packaging}");
+				var md5File = artifactFile + ".md5";
+				var md5 = File.Exists(md5File) ? File.ReadAllText(md5File) : string.Empty;
```

Comment:
A simple mechanism that reads the file if it exists. It should exist because if it didn't download, then it would be generated, and if it failed to generate, then the exception would have stopped the execution.